go/libraries/doltcore/env/actions/clone.go

```diff
@@ -215,25 +215,25 @@ func CloneRemote(ctx context.Context, srcDB *doltdb.DoltDB, remoteName, branch s
 	// create remote refs corresponding to each of them. We delete all of
 	// the local branches except for the one corresponding to |branch|.
 	for _, brnch := range branches {
```

Comment:
can you change brnch to br or something more different from `branch`- someone could pretty easily screw that up in the future.